src/eva/vision/models/wrappers/backbone.py

```diff
@@ -0,0 +1,46 @@
+"""Vision backbone helper class."""
+
+from typing import Any, Callable, Dict, List
+
+import torch
+from torch import nn
+from typing_extensions import override
+
+from eva.core.models import wrappers
+from eva.vision.models.networks import BackboneModelRegistry
+
+
+class VisionBackbone(wrappers.BaseModel):
+    """Vision backbone."""
```

Comment:
maybe add some more explanation here on what this class is used for?

Reply:
now added a short docstring to this.